oarepo_ui/templates/components/Searchbar.jinja

```diff
@@ -0,0 +1,19 @@
+{# def search_link="/search"  #}
+{% set options = [
+    {
+      "key": "docs",
```

Comment:
Remove references to NR-docs, keep it just as "All records", or, even better, let devs pass the whole options array as Searchbar component prop with a default value of "All records" on `/search` endpoint